go.mod

```diff
@@ -7,5 +7,8 @@ require (
 	github.com/DataDog/gostackparse v0.5.0
 	github.com/DataDog/sketches-go v1.0.0
 	github.com/google/pprof v0.0.0-20210423192551-a2663126120b
+	github.com/google/uuid v1.3.0
+	github.com/sqreen/go-libsqreen v0.7.3
```

Comment:
Is there any reason you're not copying this into the repository? This is a core part of appsec.

Reply:
Yes, that's what I am doing for the new bindings to libddwaf: directly have them in dd-trace-go.

Reply:
Note that I am merging this PR into a temporary `appsec` branch so that I can create other pull requests enhancing it, such as the new libddwaf bindings in a separate PR (this one being already big enough ^^').